cluster-state-service/internal/features/e2e_stream_tasks.feature

```diff
@@ -0,0 +1,8 @@
+@e2e @task @stream-tasks
+Feature: Stream Tasks
+
+  Scenario: Get task stream
+    Given I start 1 task in the ECS cluster
```

Comment:
We dont necessarily need to stop the task for testing the stream right?
`When I stream all task events and start a task` should do it. This should generate 2 events - task `pending` and task `running` afaik.